independent-publisher-connectors/ShipStationIP/apiDefinition.swagger.json

```diff
@@ -373,9 +373,12 @@
   },
   "securityDefinitions": {
     "auth": {
-      "type": "basic"
+      "type": "basic_auth"

```

Comment:
@SmolSoftBoi the type property should be "basic", I meant renaming the "auth" property above to "basic_auth". Could you please make one more fix? So that "auth" at line 375 is renamed to "basic_auth" and "type" at line 376 has value "basic".